layouts/partials/scripts.html

```diff
@@ -58,6 +58,7 @@
 
 {{ if eq .Type "act" }}
 <script src="{{ "js/act-widget.bundle.gen.js" | absURL }}"></script>
+<script src="{{ "js/sva-finder.bundle.gen.js" | absURL }}"></script> <!-- TODO Only load SVA where it's needed -->
```

Comment:
Yeah, that definitely needs to happen before review. Two options:

* We introduce something like a `has_sva_finder` frontmatter key (similar to the `blog_like` one). That's the way I'd suggest if we think we might need to include an SVA finder more than once.
* Hardcode the slug.

Reply:
Thank you! I wasn't sure how to do this. I tried to match it via hugo's filename vars, but ran into weird undefined-var-errors.